language-design.md

```diff
@@ -73,8 +101,10 @@ sympathize more with the tenor of the article or the tenor of the majority of
 the commenters?
 
 **Response**
+The essence of this conflict I think is logical vs non-logical syntax. Personally, I find some solace in programming because it is completely logical. The English language is not completely logical but I still enjoy using it for exactly that reason. English is a language all the same as any programming language. We could imagine that eventually, English will be a programming language and we will be able to speak it to computers and have them understand it. But technology is not that far along yet so as of now we have to speak to computers in a more logical language. This is solely because computers think with electrical circuits and bit (purely logical) and humans think with brain cells (which leaves room for interpretation). Some people would prefer computers think more like humans, but other technical minded people may always prefer to think like computers. -Daniel
 
-
+I'm somewhat inclined to agree with the less arrogant and condescending commenters, for example the commenter who suggested that the tools of programming are no more complicated than those of any other technical profession. But there does seem to be a sense of elitism among many of them that I really can't get behind, and I can only agree with the above sentiment so far as general-purpose programming languages are concerned. I think that telling a computer what to do is something that is
+useful for everyone to be able to do, and (I can't say with full confidence since I've only been in this class for a couple of weeks, but...) I think that's where DSLs come in. So I guess I'd say I agree with the commenters that things like Quorum shouldn't be general-purpose programming languages, but I would say that good DSLs might look a lot like Quorum does. -Aaron
 
```

Comment:
It seems that a "good DSL might look a lot like Quorum" but are there places in which a programming language like this might be to close to natural language? It might be even more frustrating for users, especially novice programmers if their code doesn't work because the correct command was "repeat this 5 times" and they types "repeat 5 times" or "do this 5 times" or "continue 5 times" --- these commands may all be equivalent in English, but if they aren't equivalent to a computer, novice programmers won't be able to understand why their code won't run.